services/api/service.go

```diff
@@ -1936,12 +2014,45 @@ func (api *RelayAPI) handleSubmitNewBlock(w http.ResponseWriter, req *http.Reque
 		"payloadBytes":           len(requestPayloadBytes),
 		"isLargeRequest":         isLargeRequest,
 	})
-	// deneb specific logging
-	if payload.Deneb != nil {
+	if payload.Version >= spec.DataVersionDeneb {
+		blobs, err := payload.Blobs()
```

Comment:
the purpose of the `GetBlockSubmissionInfo` struct is so we don't need to extract each field individually and handle the error once. 
The deneb and electra fields are mainly used for logging, so we can also remove these fields and have a separate function that returns a log object enriched with the extra payload information